scripts/generate_options_schema.js

```diff
@@ -72,8 +77,10 @@ addTypeDocOptions({
                     ).map;
                 data.enum =
                     map instanceof Map
-                        ? [...map.keys()]
-                        : Object.keys(map).filter((key) => isNaN(+key));
+                        ? [...map.keys()].map(lowerCaseFirstLetter)
+                        : Object.keys(map)
+                              .filter((key) => isNaN(+key))
+                              .map(lowerCaseFirstLetter);
```

Comment:
I was proposing this:

```suggestion
                        ? [...map.values()]
                        : Object.keys(map)
                              .filter((key) => isNaN(+key))
                              .map((key) => map[key]);
```

... however, I did this locally, and it doesn't quite work because [LogLevel](https://typedoc.org/api/enums/LogLevel.html) is a numeric enum. `--logLevel verbose` is invalid, while `--logLevel Verbose` is valid, so we also can't just lowercase the first letter.

We could do:

```suggestion
                        ? [...map.values()]
                        : Object.keys(map)
                              .filter((key) => isNaN(+key))
                              .map((key) =>
                                  typeof map[key] === "number" ? key : map[key]
                              );
```

To use the values for non-numeric mapped types, which would end up using camel-case (sorta, is `jsdoc` considered more camel case than `JSDoc`?)

`LogLevel` could be made a non-numeric enum, but that's a breaking change, so would have to wait for 0.24.